lib/floki/finder.ex

```diff
@@ -12,13 +12,24 @@ defmodule Floki.Finder do
 
   @doc """
   Find elements inside a HTML tree.
+
+  Second argument can be either a selector string, a selector struct or an array of selector structs.
   """
 
-  @spec find(html_tree, binary) :: html_tree
+  @spec find(html_tree, binary|struct|[struct]) :: html_tree
```

Comment:
@hansihe I think a new type called `selector` would fit better here, like this:

```elixir
  @type selector :: %Selector{} | binary | list
```

And use this `selector` type:

```elixir
@spec find(html_tree, selector) :: html_tree
```